### ISSUE TITLE
proposal: Edit in Editor example

### ISSUE THREAD

**Issue by HiDeoo** [delucis/starlight-examples#10]
I had to make a Starlight component override for the `<EditLink/>` component to change its behavior in development mode so that the content file is opened in the user's editor. I don't think it's worth publishing a new package just for this, so I thought it could maybe be a small and interesting example.

Here is a demo in StackBlitz (this works in StackBlitz too): https://stackblitz.com/edit/github-mydczx?file=src%2Fcomponents%2FEditInEditor.astro

Any thoughts on adding a similar example? I don't mind at all if it's rejected, e.g. not useful enough, I just thought I would ask before submitting a PR or forgetting about it.

**Comment by delucis** [comment]
Definitely! The bar for admission into the examples is low, and this example is way over it 🚀
